lib/events.js

```diff
@@ -91,9 +91,9 @@ ObjectDefineProperty(EventEmitter, 'captureRejections', {
 });
 
 ObjectDefineProperty(EventEmitter, 'errorMonitor', {
-  value: kErrorMonitor,
-  writable: false,
-  configurable: true,
```

Comment:
I believe this doesn't fix the issue raised at https://github.com/googleapis/nodejs-spanner/pull/830#issuecomment-586477413. The property is still not writable since there is only a getter without a setter.

Reply:
But writing doesn't throw anymore now, it's silently ignored.

Reply:
@flarna Unless you’re not using strict mode, this should throw an error when writing?

I would just make it a writable data property rather than a non-writable one, I don’t quite see the point of why we would want to explicitly forbid writing to it, especially if that breaks use cases…

Reply:
Also, IMHO, the property may better be configurable if there are no other points to forbid that.

Reply:
Changed to a normal property. I thought it's a bad idea to allow anyone to accidentally change this property but it seems I was wrong.

Regarding configurable: I copied this over from other locations in events.js (captureRejections, kCapture).

Reply:
You could leave it nonwritable if you make it non-enumerable, however.

Reply:
Non enumerable seems wrong to me as it is a public API. If tools like extend are not copying this property it's also a problem.